keepassxc-browser/content/banner.js

```diff
@@ -107,7 +107,8 @@ kpxcBanner.create = async function(credentials = {}) {
             kpxcBanner.banner.removeChild(dialog);
         } else {
             if (ignoreCheckbox.checked) {
-                kpxc.ignoreSite([ window.top.location.href ]);
+                const ignoreUrl = window.location.href.slice(0, window.location.href.lastIndexOf('/') + 1) + '*';
```

Comment:
what happens if the URL does not end in `/`?

Reply:
Then the URL is sliced as following:
- `https://example.com/something/testpage/login?yesplease` -> `https://example.com/something/testpage/*`
- `https://example.com/something/login.html` -> `https://example.com/something/*`
- `https://example.com/something/` -> `https://example.com/something/*`
- `https://example.com` -> `https://example.com/*`